### PR TITLE
Add example .glyphs kern and ability to read it

### PR DESCRIPTION
**Pull request by rsheeter**
Tiptoing toward #308

The existing test to confirm reading glyphs2 wghtvar == read glyphs3 wghtvar verifies the field rename works as intended.

### CODE REVIEW

File: glyphs-reader/src/font.rs [hunk 1]
```diff
@@ -968,13 +971,21 @@ impl RawFont {
         Ok(())
     }
 
+    fn v2_to_v3_kerning(&mut self) -> Result<(), Error> {
+        if let Some(kerning) = self.other_stuff.remove("kerning") {
+            self.other_stuff.insert("kerningLTR".to_string(), kerning);
```
Comment (anthrotype) [inline glyphs-reader/src/font.rs:976]: amusingly in glyphsLib we ended up actually doing the opposite i.e. down-convert the kerningLTR/kerningRTL split dictionaries into a single kerning dict, like in was in glyphs v2 and in UFO.. the history of this is complicated as everything in font-land..
You may start from https://github.com/googlefonts/glyphsLib/pull/865

Comment (rsheeter) [reply glyphs-reader/src/font.rs:976]: I explicitly didn't handle the kerningRTL set yet. My intent wasthat this is valid for both v2 and v3 LTR kerning. I _think_ you are saying it is? Perhaps I should file a bug for kerningRTL.

**Review by anthrotype — APPROVED** [review]
LGTM

File: glyphs-reader/src/font.rs (hunk 1 above)
Comment (rsheeter) [reply glyphs-reader/src/font.rs:976]: #333

**Pull request merged**